src/trio/_socket.py

```diff
@@ -356,14 +356,16 @@ def socket(
         if sf is not None:
             return sf.socket(family, type, proto)
     else:
-        family, type, proto = _sniff_sockopts_for_fileno(family, type, proto, fileno)
+        family, type, proto = _sniff_sockopts_for_fileno(  # noqa: A001
+            family, type, proto, fileno
+        )
     stdlib_socket = _stdlib_socket.socket(family, type, proto, fileno)
     return from_stdlib_socket(stdlib_socket)
 
 
 def _sniff_sockopts_for_fileno(
     family: AddressFamily | int,
-    type: SocketKind | int,
+    type_: SocketKind | int,  # name shadowing builtin
```

Comment:
Don't think there's a need for a long-lasting comment here.

Reply:
Changed in 67116bc